src/js/pages/Setup.js

```diff
@@ -194,6 +197,15 @@ class Setup extends mixin(StoreMixin) {
     PreFlightStore.beginStage();
   }
 
+  getCheckboxLabel(label, tooltipContent) {
+    return (
+      <span>
+        {label}
+        <Tooltip content={tooltipContent} width={300} wrapText={true} />
```

Comment:
wanna use the RJSC for this one? i know we need to do a project wide change, but we can just start here